### DISCUSSION TITLE
plotting to many destinations

### DISCUSSION THREAD

**Discussion by shegis-nn** [madMAx43v3r/chia-plotter#497] [Ideas]
As I noticed -d flag adds copying finished plot to destination. This process is in the background (correct me if I'm wrong) so process did not wait and start plotting next plot. It is good for network location and save some extra minutes.

Is it possible to add more destination to feel more than one disk and some kind of space left checking?

**Comment by mylastore** [comment]
We have multiple harvesters with many drives per machine. They fill up at slightly different speeds but all use the same docker container to plot. In the future, we will have a problem of moving plots around for efficiency. We don’t want to, for example, launch a container to plot to drive 15 if that drive is already full in some of the machines. We don’t want to use raid zero for final directories either because each machine has so many drives. Is Chia or MadMax planning to support multiple final directories or is there a good tool? Something that checks and creates a new plot to whichever final directory still has space. Any help is appreciated. Too many machines and directories to do it manually. Using Ubuntu server, MadMax and docker.

**Reply by shegis-nn** [reply]
I think the best way is to start plotting in one process - it starts filling temp drive, and simultaneously second process (script) for something like that:
1. Check if there is new .plot file ready
2. if no: wait 30 sec
3. if yes:
 A) check first destination for free space
      o) if no, try next destination
      o) if yes, copy plot

looks easy for someone familiar with bash - so not for me ;)

**Reply by XIII39** [reply]
You can use [Plotman](https://github.com/ericaltendorf/plotman) along side the chia-plotter.
Plotman has an archiving feature (local drives or rsync to network) that will search for a drive with available space and send your plot to it. You can also stagger the plots across the drives to reduce network clogging - plotter 1 sends to drive 0 first, plotter 2 to drive 1, plotter 3 to drive 2, etc.. 

Just set `autostart_plotting = false` in the plotman.yaml so it only archives the plots

**Comment by SebMoore** [comment]
#60 goes over why this probably won't happen.
Have you considered combining all your disks into a JBOD array so it appears as one drive?

**Comment by sotasyl** [comment]
aufs with round-robin and low space control is perfect for this

**Reply by chcoxz** [reply]
aufs sounds like what I need, but I can not seem to find much written on it.

**Reply by sotasyl** [reply]
apt install aufs-dkms
http://aufs.sourceforge.net/aufs2/man.html
Example config in fstab:
none       /storage  aufs   noatime,udba=reval,sum,create=mfsrr:150000000000:3600,br=/hdd0=rw:/hdd1=rw:/hdd2=rw:/hdd3=rw...
